cssdb.json

```diff
@@ -750,7 +750,7 @@
         "link": "https://github.com/JLHwung/postcss-ic-unit"
       }
     ],
-    "vendors_implementations": 1
+    "vendors_implementations": 0
```

Comment:
Firefox 97 will be released in 3 days : https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/97